src/pages/instances/InstanceOverviewDevices.tsx

```diff
@@ -52,6 +64,53 @@ const InstanceOverviewDevices: FC<Props> = ({ instance }) => {
           role: "rowheader",
           "aria-label": "Type",
         },
+        {
+          content: (
+            <div>
+              {device.type == "disk" ? (
+                isRootDisk(device as FormDevice) ? (
+                  <>
+                    <strong>Pool: </strong>
+                    <>{device.pool}</>
+                  </>
+                ) : (
+                  <>
+                    <strong>Storage Volume:</strong>
+                    <Link
+                      to={`/ui/project/${instance.project}/storage/pool/${device.pool}/volumes/custom/${device.source}`}
+                    >
+                      {` ${device.source}`}
+                    </Link>
+                  </>
+                )
+              ) : (
+                ""
+              )}
+              {device.type === "gpu" && (
+                <>
+                  <strong>Pci: </strong>
+                  <>{device.pci}</>
+                </>
+              )}
+              {device.type === "proxy" &&
+                deviceDetailRows([
+                  `Listen: ${device.listen}`,
+                  `Connect: ${device.connect}`,
+                ])}
+              {device.type === "usb" ? `uid: ${device.uid}` : ""}
```

Comment:
there is an issue here, `device.uid` may be undefined, in that case maybe just display `-`. Same comment for the other device values, please check all of the other cases